Gems/Atom/RPI/Code/Source/RPI.Public/Culling.cpp

```diff
@@ -519,10 +570,22 @@ namespace AZ
                 cullStats.m_cameraViewToWorld = view.GetViewToWorldMatrix();
             }
 #endif //AZ_CULL_DEBUG_ENABLED
+
+            // If connected, update the occlusion views for this scene and view combination.
+            if (const auto& entityContextId = GetEntityContextIdForOcclusion(&scene); !entityContextId.IsNull())
+            {
+                AzFramework::OcclusionRequestBus::Event(
+                    entityContextId,
+                    &AzFramework::OcclusionRequestBus::Events::UpdateOcclusionView,
+                    view.GetName(),
+                    view.GetCameraTransform().GetTranslation(),
+                    view.GetWorldToClipMatrix());
+            }
```

Comment:
do we need to return inside the if branch otherwise we will be updating Umbra and Intel.